conf/airframes/tudelft/bebop2_optitrack.xml

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE airframe SYSTEM "../airframe.dtd">
+
+<airframe name="bebop2_optitrack">
+  <description>
+    Airframe file to fly a Bebop2 drone, in stable smooth flight, through waypoints using an optitrack system, such as the Cyberzoo of TUDelft. The values are tuned so that the flight speed is limited and the drone does not over-shoot the waypoints as it would in outdoor settings.
+  </description>
+
+  <firmware name="rotorcraft">
+    <target name="ap" board="bebop2"/>
+
+    <target name="nps" board="pc">
+      <module name="fdm" type="jsbsim"/>
+      <module name="udp"/>
+    </target>
+    
+    <define name="USE_SONAR" value="FALSE"/>
+
+    <!-- Subsystem section -->
+    <module name="telemetry" type="transparent_udp"/>
+    <module name="radio_control" type="datalink"/>
+    <module name="motor_mixing"/>
+    <module name="actuators" type="bebop"/>
+    <module name="imu" type="bebop"/>
+    <module name="gps" type="datalink"/>
+    <module name="stabilization" type="indi_simple"/>
+    <module name="ahrs" type="int_cmpl_quat">
+      <configure name="USE_MAGNETOMETER" value="FALSE"/>
+      <define name="AHRS_USE_GPS_HEADING" value="TRUE"/>
+    </module>
+    <module name="ins" type="extended"/>
+  </firmware>
+
+  <modules main_freq="512">
+    <module name="geo_mag"/>
+    <module name="air_data"/>
+    <module name="send_imu_mag_current"/>
```

Comment:
We are trying to move away from the modules section, can you move these to the firmware section?